src/rail/estimation/algos/deepdisc.py

```diff
@@ -496,22 +565,25 @@ def _write_temp_file(self, qp_dstn, start_idx, num_pdfs):
             The qp.ensemble to write to the temporary file
         start_idx : int
             The starting index of the block of images
-        num_pdfs : int
-            The number of pdfs in this block of images
         """
         print("Writing out this temporary ensemble to disk")
```

Comment:
I think we can remove this print statement

Reply:
Done